packages/wrangler/src/__tests__/init.test.ts

```diff
@@ -2350,12 +2379,7 @@ describe("init", () => {
 				expectedEnvironment: "test",
 				expectedCompatDate: "1987-9-27",
 			});
-			setMockFetchDashScript({
-				accountId: "LCARS",
-				fromDashScriptName: "memory-crystal",
-				environment: mockServiceMetadata.default_environment.environment,
-				mockResponse: mockDashboardScript,
-			});
+			setMockFetchDashScript(mockDashboardScript);
```

Comment:
Kind of unfortunate we still have to use this helper :(

Reply:
Its using MSW now, with a workaround for lack of FormData support.

Reply:
👍🏻